doc/content/concepts/features/cli/troubleshooting/_index.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: "Troubleshooting CLI"
 description: ""
-aliases: [/getting-started/cli/troubleshooting]
+aliases:
+  [/getting-started/cli/troubleshooting, /concepts/features/cli/troubleshooting]
```

Comment:
This alias is to the new file location not the old. It should be `the-things-stack/interact/cli/troubleshooting/`